composer.json

```diff
@@ -29,7 +29,8 @@
     },
     "require-dev": {
 
-        "phing/phing": "2.14.0"
+        "phing/phing": "2.14.0",
```

Comment:
For instructions on using composer in OpenEMR, check this out:
http://www.open-emr.org/wiki/index.php/Composer#Migrating
(note we remove the phing stuff so it's not included in the codebase)